blocks/library/image/index.js

```diff
@@ -82,40 +85,50 @@ registerBlockType( 'core/image', {
 
 	edit( { attributes, setAttributes, focus, setFocus } ) {
 		const { url, alt, caption } = attributes;
+		const updateAlt = ( newAlt ) => setAttributes( { alt: newAlt } );
+
+		const inspector = focus && (
+			<InspectorControls key="inspector">
+				<TextControl label={ __( 'Alternate Text' ) } value={ alt } onChange={ updateAlt } />
```

Comment:
Wondering if there might be a pattern we can create here for making it easier to bind the attribute and change handlers, something like what [LinkedStateMixin](https://facebook.github.io/react/docs/two-way-binding-helpers.html) (deprecated) was meant to provide.